README.md

```diff
@@ -4,6 +4,17 @@ Styled Components are incredibly useful when building an application, but the co
 
 This library enhances [`styled-components`](https://www.styled-components.com/) by allowing you to use BEM-flavored conventions when building your components.
 
+## Installation
```

Comment:
I'd like to keep this in the prose form of the rest of it, something like...

```md
## Installation

Install `styled-components-modifiers` from npm using `npm`...

```sh
npm install --save styled-components-modifiers
` ``

...or `yarn`...

```sh
yarn add styled-components-modifiers
` ``
```